docs/Administrator-Guide/Tuning-Volume-Options.md

```diff
@@ -0,0 +1,100 @@
+ | cluster.heal-wait-queue-length | Specifies the number of heals that can be queued for the parallel background self heal jobs. | 128 | 0-10000
+ | cluster.favorite-child-policy | Specifies which policy can be used to automatically resolve split-brains without user intervention. “size” picks the file with the biggest size as the source. “ctime” and “mtime” pick the file with the latest ctime and mtime respectively as the source. “majority” picks a file with identical mtime and size in more than half the number of bricks in the replica. | none | none/size/ctime/mtime/majority
+ | cluster.use-anonymous-inode | Setting this option heals directory renames efficiently | no | no/yes
+Logging | diagnostics.brick-log-level | Changes the log-level of the bricks | INFO | DEBUG/WARNING/ERROR/CRITICAL/NONE/TRACE
+ | diagnostics.client-log-level | Changes the log-level of the clients. | INFO | DEBUG/WARNING/ERROR/CRITICAL/NONE/TRACE
+ | diagnostics.latency-measurement | Statistics related to the latency of each operation would be tracked. | Off | On/Off
+ | diagnostics.dump-fd-stats | Statistics related to file-operations would be tracked. | Off | On/Off
+Performance | *features.trash | Enable/disable trash translator | off | on/off
+ | *performance.readdir-ahead | Enable/disable readdir-ahead translator in the volume |	off | on/off
+ | *performance.read-ahead | Enable/disable read-ahead translator in the volume | off | on/off
+ | *performance.io-cache | Enable/disable io-cache translator in the volume | off | on/off
+ | features.read-only | Enables you to mount the entire volume as read-only for all the clients (including NFS clients) accessing it. | Off | On/Off
+ | performance.write-behind-window-size | Size of the per-file write-behind buffer. | 1MB | Write-behind cache size
+ | performance.io-thread-count | The number of threads in IO threads translator. | 16 | 1-64
+ | performance.flush-behind | If this option is set ON, instructs write-behind translator to perform flush in background, by returning success (or any errors, if any of previous writes were failed) to application even before flush is sent to backend filesystem. | On | On/Off
+ | performance.cache-max-file-size | Sets the maximum file size cached by the io-cache translator. Can use the normal size descriptors of KB, MB, GB,TB or PB (for example, 6GB). Maximum size uint64. | 2 ^ 64 -1 bytes | size in bytes
+ | performance.cache-min-file-size | Sets the minimum file size cached by the io-cache translator. Values same as "max" above | 0B | size in bytes
+ | performance.cache-refresh-timeout | The cached data for a file will be retained till 'cache-refresh-timeout' seconds, after which data re-validation is performed. |	1s | 0-61
+ | performance.cache-size | Size of the read cache. | 32 MB | size in bytes
+ | geo-replication.indexing | Use this option to automatically sync the changes in the filesystem from Master to Slave. | Off | On/Off
```

Comment:
Primary and secondary instead of master..slave.

Reply:
Done